pkgs/build-support/rust/hooks/default.nix

```diff
@@ -86,6 +91,7 @@
       ];
       substitutions = {
         inherit (rust.envVars) rustTargetPlatformSpec setEnv;
+        pythonInterpreter = lib.versions.majorMinor pythonOnBuildForHost.version;
```

Comment:
Pretty sure this is the wrong way since we build python packages for multiple python versions